packages/bundler-plugin-core/src/utils/Output.ts

```diff
@@ -11,9 +11,26 @@ import { type NormalizedOptions } from "./normalizeOptions.ts";
 import { detectProvider } from "./provider.ts";
 import { uploadStats } from "./uploadStats.ts";
 
+interface InternalOptions {
+  frozenBundleName: boolean;
+  frozenPluginDetails: boolean;
+}
+
+interface SetPluginOptions {
+  frozen?: boolean;
+}
+
+interface SetBundleNameOptions {
+  frozen?: boolean;
+}
+
 class Output {
   userOptions: NormalizedOptions;
```

Comment:
Ideally we never store this anymore, and instead turn this into properties on the class.

Reply:
Done